templates/taxi/manufacturer_form.html

```diff
@@ -8,5 +8,6 @@ <h1>{{ object|yesno:"Update,Create" }} manufacturer</h1>
     {{ form|crispy }}
 
     <input type="submit" value="Submit" class="btn btn-primary">
+    <a href="{% url 'taxi:manufacturer-list' %}" class="btn btn-secondary">Cancel</a>
```

Comment:
According to the checklist, it's recommended to use a 2 spaces indentation in your `.html` files. You are currently using 4 spaces. Please, adjust your indentation to improve your code readability and maintain consistency.